pkg/ui/react-app/src/pages/graph/Panel.tsx

```diff
@@ -333,6 +344,14 @@ class Panel extends Component<PanelProps & PathPrefixProps, PanelState> {
   };
   handleChangeForceTracing = (event: React.ChangeEvent<HTMLInputElement>): void => {
     this.setOptions({ forceTracing: event.target.checked });
+
+    if (event.target.checked && this.state.traceId) {
+      this.setState({ hoverMessage: `TraceID: ${this.state.traceId}` });
+    } else if (event.target.checked) {
+      this.setState({ hoverMessage: 'TraceID is not available. Enable it in your options.' });
```

Comment:
Won't this show up if the option is checked but a query hasn't been executed yet? :thinking:

Reply:
I will look into it more deeply 👍